rboot-sampleproject/main.c

```diff
@@ -18,9 +18,6 @@
 
 static os_timer_t network_timer;
 
-void ICACHE_FLASH_ATTR user_rf_pre_init(){

```

Comment:
This was causing a duplicate definition compilation error w/ the latest open sdk. If you think it's necessary, I can figure out a way to make it happy w/ or w/o.